content/billing-pricing/pricing.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Pricing
+weight: 2
+---
+
+## Pay as you go pricing
+
+Pay as you go pricing allows you to pay for only what you use. 
+
+Currently it is only possible to run one build at a time with pay as you go.
+
+### Instance Types
```

Comment:
I feel like the instance types information would be more suited to this page: https://docs.codemagic.io/specs/machine-type/

Reply:
Instance types should be shown on this page so people can see what the price relates to without having to jump to another page.